interface/src/app/analyze/analysis/sampleBin.tpl.html

```diff
@@ -1,10 +1,9 @@
-<svg id="sample-bin" width="100" height="60" ng-click="show()">
-  <image x="27" y="6" width="45" height="45"
-      xlink:href="static/assets/testtube-expt2.svg">
-  </image>
-  <g ng-if="sb.heatmapData.samples.length">
-    <text text-anchor="left" x="28" y="43">&times;</text>
-    <text text-anchor="left" x="44" y="43">{{
-      sb.heatmapData.samples.length }}</text>
-  </g>
-</svg>
+<li ui-sref-active="active" class="sample-bin">
+  <a href ui-sref="analysis-detail">
```

Comment:
Hmm. Good question! This follows the pattern of all the other menu items found in `src/index.html`. Looking up the documentation...

Okay, take a look at the examples on Bootstrap for `navbar`s: https://getbootstrap.com/components/#navbar . All of the `<a>` tags have `href='#'` as an attribute. Strictly speaking, that attribute is optional (https://www.w3.org/MarkUp/1995-archive/Elements/A.html ), but without it the `<a>` tag is not a link but merely a text anchor and I suspect the browser will not behave the way we need it to for a menu.